src/components/users/login-form.scss

```diff
@@ -0,0 +1,69 @@
+@import "../../style/common/constants";
+
+.login-form {
+    padding-top: 1em;
+  }
+  
+.login-form__text {
+    font-family: $font-family-regular;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 1em;
+    line-height: 1em;
+}
+
+.login-form__input {
+    width: 18em;
+    height: 2em;
+}
+
+.login-form__btn-text {
+    width: 15.5em;
+    height: 2em;
+    border: 0;
```

Comment:
A good way to go regarding to style is keep it sorted alphabetically. Just a suggestion for later on.